### PR TITLE
JSAEM 2-28 feat: add basic snapshot test

### PR DESCRIPTION
**Pull request by MichaelCTH**

### CODE REVIEW

File: src/Navigator/__test__/__snapshots__/TabBarIcon.test.jsx.snap [hunk 1]
```diff
@@ -0,0 +1,3 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`<TabBarIcon /> render 1`] = `<Text />`;
```
Comment (ikarasz) [inline src/Navigator/__test__/__snapshots__/TabBarIcon.test.jsx.snap]: this looks pretty strange to me, it is only a Text component? Is the icon set via css or what?

File: .eslintrc.json [hunk 2]
```diff
@@ -17,5 +17,6 @@
     "sourceType": "module"
   },
   "plugins": ["react"],
+  "ignorePatterns":["*.test.jsx"],
```
[the remark this answers is not in the document]
Comment (MichaelCTH) [reply .eslintrc.json]: fixed

File: src/Navigator/__test__/__snapshots__/TabBarIcon.test.jsx.snap (hunk 1 above)
Comment (ikarasz) [reply src/Navigator/__test__/__snapshots__/TabBarIcon.test.jsx.snap]: https://github.com/expo/expo/issues/3566

Comment (ikarasz) [reply src/Navigator/__test__/__snapshots__/TabBarIcon.test.jsx.snap]: the test file could go right into the component folder, it doesn't needs to be in a `__test__` folder

Comment (MichaelCTH) [reply src/Navigator/__test__/__snapshots__/TabBarIcon.test.jsx.snap]: fixed

File: .eslintrc.json [hunk 3]
```diff
@@ -2,7 +2,8 @@
   "env": {
     "browser": true,
     "es6": true,
-    "node": true
+    "node": true,
+    "jest": true
```
Comment (ikarasz) [inline .eslintrc.json]: Sorry my bad, I didn't notice this the last time. As I mentioned in the first review, you can use overrides in the eslint. So you can specify rules for specific folders/files. With this you can say that only the `.test.js` files have the jest environment. If you don't do this, then eslint will let us use the describe global method in any source file which of course not exists.

Comment (MichaelCTH) [reply .eslintrc.json]: fixed

**Pull request merged**